Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ruby:3.0
+FROM ruby:3.2.2
```

Comment:
same ... verision `3.2` ... or is there a special reason why we should use minor version here?

Reply:
No, no reason why defining special minor version here, I changed it.